modules/planning/common/path/path_data.h

```diff
@@ -36,12 +36,11 @@ namespace planning {
 class PathData {
  public:
   enum class PathPointType {
-    UNKNOWN,
-    OFF_ROAD,
     IN_LANE,
     OUT_ON_FORWARD_LANE,
     OUT_ON_REVERSE_LANE,
-    BACK_TO_IN_LANE,
+    OFF_ROAD,
+    UNKNOWN,
```

Comment:
Just a reminder that be careful about changing enum value. It breaks if some existing code is using its integer value, and the implicite default value which is the first entry.

Reply:
Thanks. I just wanted to change the sequence to make it more meaningful and didn't add/delete anything.

Reply:
Thanks. This enum has not been used yet. I talked with Jinyun and we both agreed to delete this BACK_TO_IN_LANE one.